Error: Cannot find module './path.cjs' while building on next-i18next v9.0.0

Error: Cannot find module './path.cjs' while building

![image](https://user-images.githubusercontent.com/25684433/141442374-4d208474-1c55-4c26-8c4e-c58f0663c733.png)


next-i18next 9.0.0 & 8.10.0 & 8.9.0
yarn 1.22.10
next 11.1.2
node 14.18.0

## I try to fix it


I've rename "path.cjs" to "path.js" at readFile.js at i18next-fs-backend package directory in node_modules then it worked! 🤔

![image](https://user-images.githubusercontent.com/25684433/141441701-cd12c42e-dc09-4f83-a2ce-ca7740ab7be1.png)

@isaachinman
```
Module not found: Can't resolve 'fs' in '/nextjs/node_modules/next-i18next/dist/commonjs'
Native Node.js APIs are not supported in the Edge Runtime with `concurrentFeatures` enabled. Found `fs` imported.
```

--- a/package.json
+++ b/package.json
@@ -114,5 +114,8 @@
   "peerDependencies": {
     "next": ">= 10.0.0",
     "react": ">= 16.8.0"
+  },
+  "resolutions": {
+    "i18next-fs-backend": ">=1.1.4"
   }
 }

